addons/l10n_ar/i18n/l10n_ar.pot

```diff
@@ -4,10 +4,10 @@
 #
```

Comment:
Please  be careful in the po/pots. We have +2400 lines of diff in each.  Is this necessary?  Are we doing a cleanup in the meantime?

Reply:
If it is a cleanup, maybe better in a separate commit.

Reply:
Hi @jco-odoo 
This is because it seems that in master the account.account name, the account.tax and account.group name and description, the account.template name are translatable fields, etc

So when I run the exportation translation wizard all this new translations were added.
I will added in a separate commit as you requested.

Best Regards

Reply:
Just don't install l10n_multilang I guess

Reply:
@zaoral

Reply:
oh... I did not know that.
I will do that and simplify the unit tests.
Thanks Jos